bin/znapzendzetup

```diff
@@ -482,7 +482,7 @@ sub main {
     }
     print STDERR <<NOTE;
 NOTE: if you have modified your configuration, send a HUP signal
-(pkill -HUP znapzend) to your znapzend daemon for it to notice the change.
+(sudo launchctl kill -HUP system/org.znapzend) to your znapzend daemon for it to notice the change.
```

Comment:
This text might now confuse non mac users, as launchctl is MacOS specific

Reply:
You're right. I'm afraid I don't know how to insert a conditional statement, to check. Like I said, I'm not a programmer.
TIA
Xen

Reply:
you could replace the code with:

```perl
if ($^O eq 'darwin') {
   print STDERR <<NOTE
... 
NOTE
} else {
   print STDERR <<NOTE
...
NOTE
}
```